sapp/ui/filters.py

```diff
@@ -78,8 +78,17 @@ def all_filters(session: Session) -> List[Filter]:
 
 
 def save_filter(session: Session, filter: Filter) -> None:
-    LOG.debug(f"Storing {filter}")
-    session.add(FilterRecord.from_filter(filter))
+
+    existing = session.query(FilterRecord).filter(FilterRecord.name == filter.name).first()
+    
+    if not existing:
+        session.add(FilterRecord.from_filter(filter))
+        LOG.debug(f"Adding {filter}")
+    else:
+        existing.description = filter.description
+        existing.json = filter.json
```

Comment:
don't we need to call `existing.save()`?

Reply:
I think it saves by default!

Reply:
My bad, I missed the `session.commit()` below.